@stellar/design-system/src/global.scss

```diff
@@ -23,6 +25,7 @@
   --pal-text-secondary: rgba(var(--pal-text-base-rgb), 0.6);
   --pal-text-tertiary: rgba(var(--pal-text-base-rgb), 0.4);
   --pal-text-link: #5332e6;
+  --pal-text-link-hover: #241080;
 
```

Comment:
may not be necessary, but since it's repeated for`--pal-brand-primary-hover-rgb` and ` --pal-text-link-hover`,  would it help to set `#241080` as a var?

Reply:
It might be more clear to keep them separate as they are part of the color palette. Thanks for pointing it out, though!